### PR TITLE
Disable recurring details while editing the cancelled appointment

### PR DESCRIPTION
**Pull request by AlekhyaYalla**

### CODE REVIEW

File: ui/app/appointments/views/manage/newAppointment.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h2 ng-show="isEditMode()">{{ ::'EDIT_APPOINTMENT_KEY' | translate}}</h2>
                             <li ng-repeat="appProvider in appointment.providers" ng-show="appProvider.response != 'CANCELLED'">
                                 <label>{{ appProvider.name || appProvider.person.display }}</label>
                                 <input type="button" value="{{'APPOINTMENT_PROVIDER_REMOVE_ACTION' | translate }}" ng-click="removeProviderFromAttendees(appProvider)"
-                                 ng-disabled="isEditMode() && !isEditAllowed()"/>
+                                       ng-disabled="isEditMode() && !isEditAllowed()"/>
```
Comment (bsneha90) [inline ui/app/appointments/views/manage/newAppointment.html]: isEditMode() && !isEditAllowed() , can we bring this out in a common variable like "let isEditDisabled = isEditMode() && !isEditAllowed()"

File: ui/app/appointments/views/manage/newAppointment.html [hunk 2]
```diff
@@ -129,7 +129,7 @@ <h2 ng-show="isEditMode()">{{ ::'EDIT_APPOINTMENT_KEY' | translate}}</h2>
 
                 <p class="checkbox-container">
                     <label>
-                        <input type="checkbox" ng-model="appointment.setRecurring">
+                        <input type="checkbox" ng-model="appointment.setRecurring" ng-disabled="isEditMode() && !isEditAllowed()">
```
Comment (bsneha90) [inline ui/app/appointments/views/manage/newAppointment.html]: same can we use "isEditDisabled" ?

Comment (AlekhyaYalla) [reply ui/app/appointments/views/manage/newAppointment.html]: yes , will make changes and update

File: ui/app/appointments/views/manage/newAppointment.html (hunk 1 above)
Comment (AlekhyaYalla) [reply ui/app/appointments/views/manage/newAppointment.html]: Added changes

**Pull request merged**